package.json

```diff
@@ -23,6 +23,7 @@
   },
   "devDependencies": {
     "browserify": "^10.0.0",
+    "es6-promise": "^2.3.0",
```

Comment:
The tests were broken (even without any changes) for browserify > 10.2.4 the change that I made was to make sure the tests were run sequentially without enter into race conditions which happened to fix the issues. I haven't change any other test, just make sure they were properly run in sequence